frontend/src/components/ParticipantForm.js

```diff
@@ -96,6 +96,32 @@ const ParticipantForm = ({
           />
         </FormControl>
       </Grid>
+      <Grid item xs={12} sm={6}>
```

Comment:
This was not in the ticket, but while reviewing this, the inputs get minuscule before switching to the mobile view when the sidebar is open.

can you remove the `sm={6}` on each input that has it , and replace it with `md={6}`?
can you also do the same to the `VisitForm.js`? Don't forget to update the snapshot!

Reply:
I addressed the issue.